quartermaster/src/main/java/com/github/reflectoring/infiniboard/quartermaster/widget/WidgetController.java

```diff
@@ -22,23 +25,25 @@
 @RequestMapping("/api/widgets")
 public class WidgetController {
 
-    private WidgetConfigService widgetService;
-    private WidgetConfigMapper  widgetConfigMapper;
-    private SourceDataMapper    sourceDataMapper;
+    private WidgetConfigRepository widgetConfigRepository;
+    private WidgetConfigMapper     widgetConfigMapper;
+    private SourceDataMapper       sourceDataMapper;
+    private SourceDataRepository   sourceDataRepository;
 
     @Autowired
-    public WidgetController(WidgetConfigService widgetService,
+    public WidgetController(WidgetConfigRepository widgetConfigRepository,
+                            SourceDataRepository sourceDataRepository,
                             WidgetConfigMapper widgetConfigMapper,
                             SourceDataMapper sourceDataMapper) {
-
-        this.widgetService = widgetService;
+        this.widgetConfigRepository = widgetConfigRepository;
+        this.sourceDataRepository = sourceDataRepository;
         this.widgetConfigMapper = widgetConfigMapper;
         this.sourceDataMapper = sourceDataMapper;
     }
 
     @RequestMapping(value = "/{widgetId}", method = GET)
-    public ResponseEntity<HalJsonResource> getWidget(@PathVariable String widgetId) {
-        WidgetConfig widgetConfig = widgetService.loadWidget(widgetId);
+    public ResponseEntity<HalJsonResource> getOne(@PathVariable String widgetId) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)